### ISSUE TITLE
Incorrect display Total Objects and Cache size on Cache Manager

### ISSUE THREAD

**Issue by CimborNet** [PHPfox-Official/phpfox-v4-issues#2532]
## What's happened?
If you look at the cache cleaning page, then we see not a correct display of the number of objects in the cache and the size of the cache.

## Steps to reproduce:
1. AdminCP
2. Maintenance => Cache Manager
3. Look at the values before and after clicking on the clear cache button.

## What's expected?
It is expected that after clearing the cache of objects in the cache will be less than before cleaning it.
And second: The size of the cache indicates the version of the program, and not its actual size.

## Browsers and Devices tested
All

## Server information
PHP Version: 7.2.8
Operating System: Linux
GZIP: Enabled
SQL Driver Version: MySQL 5.6.34
SQL Slave Enabled: No
SQL Total Slaves: N/A
SQL Slave Server: N/A

## phpFox version
Core Version: 4.7.2

## Screenshots
![3](https://user-images.githubusercontent.com/11414490/50709655-cb011600-1068-11e9-9a65-1d970ab01a24.jpg)

**Comment by harrison05** [comment]
I don't think this is a bug because, before clear cache, cache objects were minimized and reduced total objects in the cache system. So, after you clear cache, object rewrites again and it only optimizes after users access to any page in Site. 
It weird but working as design.

Regards.

**Comment by spirogg** [comment]
@harrison05 @CimborNet is correct the cache size shows the version number not the actual cache size

- ![capture](https://user-images.githubusercontent.com/17736688/50720910-25f03880-107c-11e9-96fe-1215d1c15954.PNG)

**Comment by CimborNet** [comment]
This is not entirely true.

The fact is that before cleaning the server worked for at least 24 hours. And during these 24 hours, 10 objects were collected?

Then the next step:
- I close access to the server for users.
- Clearing the site cache.

And suddenly there are already 244 objects? Kind of weird. No one enters the site. No activity on the site.
Only cleaning the cache and that's it. The data is completely different.

It is logical to expect that before cleaning the cache there should be more objects in the cache than after cleaning. Especially since before the cleaning there was an intense load on the server from the users.

In other words, to say that with an intensive load of 10 objects, and when cleaning the cache immediately 244 (and this is with zero activity, the server is closed for users)

Yes, indeed, on other my cache systems, the cache indicates 4m and 230k objects. But 4m> 230k
But in this case it is not clear, it’s all objects or their shortened value.
